.env

```
@@ -0,0 +1,4 @@
+VITE_APPWRITE_PROJECT_ID = "6923557e003c59f77f8b"
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)